dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -629,12 +629,14 @@ private static CompletionsOptions CreateCompletionsOptions(string text, OpenAIPr
 
             if (message.AdditionalProperties?.TryGetValue(NameProperty, out string? name) is true)
             {
-                azureMessage.Name = name;
-
                 if (message.AdditionalProperties?.TryGetValue(ArgumentsProperty, out string? arguments) is true)
                 {
                     azureMessage.FunctionCall = new FunctionCall(name, arguments);
                 }
+                else
```

Comment:
Is there a test case that failed before this change and now succeeds?

Reply:
Looks like our test case was made to match the incorrect behavior, which is why it was succeeding but now failing with this PR. I need to update the test.